### PR TITLE
Add test code, remove some HTTP headers 

### PR DESCRIPTION
**Pull request by Mineru98**

### CODE REVIEW

File: src/threads-api.ts [hunk 1]
```diff
@@ -30,19 +30,6 @@ export class ThreadsAPI {
...
-    'sec-ch-ua-platform-version': '"13.0.0"',
-    'sec-fetch-dest': 'empty',
-    'sec-fetch-mode': 'cors',
-    'sec-fetch-site': 'same-origin',
-    'user-agent':
-      'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36',
-    'viewport-width': '1728',
     'x-asbd-id': '129477',
     'x-fb-lsd': 'NjppQDEgONsU_1LCzrmp6q',
     'x-ig-app-id': '238260118697367',
```
Comment (junhoyeo) [inline src/threads-api.ts:35]: Can't we remove this(`x-asbd-id`, `x-fb-lsd`, and `x-ig-app-id`) and add those headers back?
context: https://github.com/junhoyeo/threads-api/pull/4#discussion_r1254573262

**Review by junhoyeo — APPROVED** [review]
tests looks great to me, thanks!

File: src/threads-api.ts (hunk 1 above)
Comment (Mineru98) [reply src/threads-api.ts:35]: If we were to delete that key now, the library would not work at all.

Comment (junhoyeo) [reply src/threads-api.ts:35]: gotcha, I'll just merge this PR for now then.

**Pull request merged**